BlueChip/chain.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "BlueChip",
+    "status": "live",
+    "website": "https://www.bluechip.link",
+    "network_type": "mainnet",
+    "pretty_name": "BlueChip",
+    "chain_id": "bluechip_1",
+    "bech32_prefix": "bcp",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips

Reply:
Do I simply just add BlueChip's information to the chart?

Reply:
yes